content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -155,33 +162,41 @@ int CalculateTestSquare( const int number)
 
 TEST(MathTests, FastSquare_Integer_Squared1)
 {
+    // despite the name suggesting this verifies the math of computing a square
+    // this test trusts the function is correct, and only proves an identity property
```

Comment:
I don't think the first line is necessary. Other than that, I have some suggestions on the wording for this and the tests below, which includes omitting the word "only"
```suggestion
    // Assumes the function works correctly and proves an identity property
```

Reply:
Yep, I like these changes